test/functional/page_objects/settings_page.ts

```diff
@@ -1059,4 +1059,13 @@ export class SettingsPageObject extends FtrService {
       [data-test-subj="indexPatternOption-${newIndexPatternTitle}"]`
     );
   }
+
+  async createNewDataView(dataViewName: string) {
+    await this.testSubjects.click('createDataViewButton');
+    await this.testSubjects.setValue('createIndexPatternTitleInput', dataViewName, {
+      clearWithKeyboard: true,
+      typeCharByChar: true,
+    });
+    await this.testSubjects.click('saveIndexPatternButton');
+  }
```

Comment:
`settings_page` is for actions on Stack Management pages. We can't use it as a replacement for existing discover tests as it intended to use UnifiedSearch actions.

Reply:
I was thinking to create a `DataView` FTR service for sake of not copying the data view actions code across Discover/UnifiedSearch/Settings pages. Does it makes sense?